### DISCUSSION TITLE
Question on releases

### DISCUSSION THREAD

**Discussion by msalehisedeh** [inversify/InversifyJS#1573] [General]
Hi,
I noticed this library and wanted to incorporate it into my project. But I've noticed there has not been any major release for a couple of years. This is concerning me and I was wondering if there is any reason behind it.

**Comment by PodaruDragos** [comment]
hey, the quick answer is there is no one at the moment to release. 
In the long run i think `inversify` is a battle tested package. 
Since it has no dependencies there is no need to worry about vulnerabilities.

As per releases, it will come a time when someone with release rights will come back and do releases.
I suggest you try it out tho, see how it goes for your project and decide it after.

**Comment by rsaz** [comment]
Inversify is a robust, production-ready, and widely adopted IoC (Inversion of Control) framework for Node.js. Among IoC libraries, it's one of the most feature-complete—far ahead of alternatives like Microsoft's TSyringe and others.

For those who may have noted a lull in releases, this is actively changing. @notaphplover and I are now dedicated core maintainers, and we’ve mapped out a roadmap to evolve and enhance Inversify with new features.

If you're considering Inversify, it’s an excellent choice—trusted in production, actively maintained, and designed to offer an unparalleled IoC experience.

PS: Inversify also powers the [ExpressoTS Framework](https://expresso-ts.com/).
